GettingStarted.md

```diff
@@ -14,7 +14,8 @@ This page is a quick start for working with seL4 and its ecosystem.
 ### Sources
 
 Code and proofs are available on [GitHub](https://github.com/seL4), under standard
-[open-source licenses](http://sel4.systems/Info/GettingStarted/license.pml).
+[open-source licenses](https://github.com/seL4/seL4/tree/master/LICENSES), 
```

Comment:
Maybe make this a bit more generic?
```suggestion
open-source licenses. Details can be found in the ``LICENSE.md`` file of each repository, 
e.g. [here](https://github.com/seL4/seL4/blob/master/LICENSE.md) for the seL4 kernel.
```

Reply:
Bases on the remark from @lsf37 this could become:
```suggestion
open-source licenses. Details can be found in the ``LICENSES`` folder of each repository
(e.g. [here](https://github.com/seL4/seL4/tree/master/LICENSES) for the seL4 kernel) and
and in the [SPDX](https://spdx.dev) tags in each file
```